pkg/driverbuilder/builder/builders.go

```diff
@@ -76,7 +77,6 @@ type commonTemplateData struct {
 	ModuleFullPath    string
 	BuildModule       bool
 	BuildProbe        bool
-	GCCVersion        string
```

Comment:
We don't need the templated GCCVersion anymore since we already pass a correctly formatted cmake command to builders.